rhel8/profiles/ccc.profile

```diff
@@ -0,0 +1,261 @@
+#    - audit_rules_privileged_commands_crontab
+#    - audit_rules_mac_modification
+#    - audit_rules_session_events
+#    - audit_rules_privileged_commands_at
+    - directory_access_var_log_audit
+#    - audit_rules_kernel_module_loading_insmod
+#    - audit_rules_kernel_module_loading_rmmod
+#    - audit_rules_kernel_module_loading_modprobe
+#    - audit_rules_kernel_module_loading_init
+#    - audit_rules_kernel_module_loading_finit
+#    - audit_rules_kernel_module_loading_delete
+
+    # Configuration (all are defaults)
+#    - var_auditd_data_retention=incremental_async  # typo? var_auditd_flush used instead
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+
+    # audispd plugins
+    - auditd_audispd_syslog_plugin_activated
+
+    ### Services
+
+    # sshd
+    - sshd_disable_root_login
+    - sshd_enable_strictmodes
+    - disable_host_auth
+    - sshd_disable_user_known_hosts
+    - sshd_disable_rhosts
+    - sshd_disable_empty_passwords
+    - sshd_disable_kerb_auth
+    - sshd_disable_gssapi_auth
+    - sshd_idle_timeout_value=10_minutes
+#    - sshd_set_idle_timeout  # duplicate, see below
+    - sshd_set_keepalive
+    - sshd_enable_warning_banner
+    - sshd_disable_rhosts_rsa
+
+    # ssh client settings
+
+    # Firewalld
+    - service_firewalld_enabled
+
+    # abrt
+    - package_abrt_removed
+
+    # Chrony
+
+    # rngd
+
+    # SSSD
+
+    # Crypto settings
+
+    # Libreswan
+
+    ### User sessions
+
+    # Login
+    - no_empty_passwords
+    - var_accounts_passwords_pam_faillock_deny=3
+    - accounts_passwords_pam_faillock_deny
+    - var_accounts_passwords_pam_faillock_fail_interval=900
+    - accounts_passwords_pam_faillock_interval
+    - var_accounts_passwords_pam_faillock_unlock_time=never
+    - accounts_passwords_pam_faillock_unlock_time
+    - dconf_gnome_banner_enabled
+    - login_banner_text=usgcb_default
+    - dconf_gnome_login_banner_text
+    - disable_users_coredumps
+    - var_accounts_max_concurrent_login_sessions=10
+    - accounts_max_concurrent_login_sessions
+    - securetty_root_login_console_only
+    - var_password_pam_unix_remember=5
+    - accounts_password_pam_unix_remember
+
+    # Password
+#    - no_empty_passwords   # duplicate in Login
+    - var_accounts_password_minlen_login_defs=12
+    - accounts_password_minlen_login_defs
+    - var_password_pam_minlen=12
+    - accounts_password_pam_minlen
+    - var_password_pam_ocredit=1
+    - accounts_password_pam_ocredit
+    - var_password_pam_dcredit=1
+    - accounts_password_pam_dcredit
+    - var_password_pam_ucredit=1
+    - accounts_password_pam_ucredit
+    - var_password_pam_lcredit=1
+    - accounts_password_pam_lcredit
+    - var_password_pam_difok=4
+    - accounts_password_pam_difok
+    - var_password_pam_maxrepeat=3
+    - accounts_password_pam_maxrepeat
+    - var_password_pam_maxclassrepeat=4
+    - accounts_password_pam_maxclassrepeat
+
+    # umask
+    - accounts_umask_etc_profile
```

Comment:
Please set value variable `var_accounts_user_umask` because it is used in rule `accounts_umask_etc_profile`.

Reply:
The default value is "027" and is what the profile expects. But there should be no harm in selecting it explicitly.

Reply:
Great!

My opinion is to set them always explicitly, because:

* the default might change which would change the profiles that rely on default, which might happen without noticing
* it is harder to find the actual value if the variable is not set in profile yaml, because you have to first realize that the rule is parametrized, then find the name of the variable, then find then variable yaml file, then open it, ...

I would even say that in future we should have a test or a build-time check that enforces explicit settings.